content/applications/essentials/contacts.rst

```diff
@@ -0,0 +1,243 @@
+Set the :guilabel:`Job Position`, which appears if the :guilabel:`Contact` address type has been
+selected. This is similar to the :guilabel:`Individual` contact.
+
+To add a note, click on the text field next to :guilabel:`Notes`, and write anything that is
+applicable to the customer or contact.
+
+Then, click :guilabel:`Save & Close` to save the address, and close the :guilabel:`Create Contact`
+window. Or, click :guilabel:`Save & New` to save the address, and immediately input another one.
+
+Sales & Purchase tab
+--------------------
+
+Next, is the :guilabel:`Sales & Purchases` tab, which only appears when the *Sales*, *Purchase*,
+**or** *Point of Sale* applications are installed.
+
+Below that, the :guilabel:`Fiscal Position` can be set.
+
+.. tabs::
+
+   .. tab:: Sales section
+
+      Under the :guilabel:`Sales` heading, a specific :guilabel:`Salesperson` can be assigned to a
+      contact. To do that, click the :guilabel:`Salesperson` drop-down field, and select one. Create
+      a new :guilabel:`Salesperson` by typing the user's name, and making the appropriate selection.
+
+      Certain :guilabel:`Payment Terms`, or a certain :guilabel:`Pricelist`, can also be set, if
+      needed. Click the drop-down menu next to :guilabel:`Payment Terms`, and change it to one of
+      the preselected :guilabel:`Payment Terms`, or :guilabel:`Create` a new one. Select the
+      :guilabel:`Pricelist` drop-down menu to choose the appropriate :guilabel:`Pricelist`.
+
+      Click into the :guilabel:`Delivery Method` field to select an option from the drop-down menu.
+
+   .. tab:: Point Of Sale section
+
+      Under the :guilabel:`Point Of Sale` heading, enter a :guilabel:`Barcode` that can be used to
+      identify the contact. Use the :guilabel:`Loyalty Points` field to track points the user won as
+      part of a *Loyalty Program*.
+
+   .. tab:: Purchase section
+
+      Specify :guilabel:`Payment Terms`, :guilabel:`1099 Box` information, and a preferred
+      :guilabel:`Payment Method` here. A :guilabel:`Receipt Reminder` can be set here, as well.
+
+   .. tab:: Misc section
+
+      Under the :guilabel:`Misc.` heading, use :guilabel:`Reference` field to add any additional
+      information for this contact. If this contact should only be accessible for one company in a
+      multi-company database, select it from the :guilabel:`Company` field drop-down list. Use the
+      :guilabel:`Website` drop-down menu to restrict the publishing of this contact to one website
+      (if working on a database with multiple websites). Select one or more :guilabel:`Website Tags`
+      to assist in filtering published customers on the `/customers` website page. Select an
+      :guilabel:`Industry` for this contact from the drop-down menu. Use the :guilabel:`SLA
+      Policies` field to assign a *Helpdesk* SLA policy to this contact.
+
+Under the :guilabel:`Fiscal Information` heading, select a :guilabel:`Fiscal Position` from the
+drop-down menu.
```

Comment:
I dont think using tabs here is very effective, here is my reasoning. [Tab headings cannot be translated](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#content-tabs), as well the information inside hidden tabs cannot be searched for (using `ctrl + f` in browser). I think all of this information is important to show up-front. So suggesting to use regular-ole' sections here :)
```suggestion
Sales section
~~~~~~~~~~~~~

Under the :guilabel:`Sales` heading, a specific :guilabel:`Salesperson` can be assigned to a
contact. To do that, click the :guilabel:`Salesperson` drop-down field, and select one. Create a new
:guilabel:`Salesperson` by typing the user's name, and making the appropriate selection.

Certain :guilabel:`Payment Terms`, or a certain :guilabel:`Pricelist`, can also be set, if needed.
Click the drop-down menu next to :guilabel:`Payment Terms`, and change it to one of the preselected
:guilabel:`Payment Terms`, or :guilabel:`Create` a new one. Select the :guilabel:`Pricelist`
drop-down menu to choose the appropriate :guilabel:`Pricelist`.

Click into the :guilabel:`Delivery Method` field to select an option from the drop-down menu.

Point Of Sale section
~~~~~~~~~~~~~~~~~~~~~

Under the :guilabel:`Point Of Sale` heading, enter a :guilabel:`Barcode` that can be used to
identify the contact. Use the :guilabel:`Loyalty Points` field to track points the user won as part
of a *Loyalty Program*.

Purchase section
~~~~~~~~~~~~~~~~

Specify :guilabel:`Payment Terms`, :guilabel:`1099 Box` information, and a preferred
:guilabel:`Payment Method` here. A :guilabel:`Receipt Reminder` can be set here, as well.

Misc section
~~~~~~~~~~~~

Under the :guilabel:`Misc.` heading, use :guilabel:`Reference` field to add any additional
information for this contact. If this contact should only be accessible for one company in a
multi-company database, select it from the :guilabel:`Company` field drop-down list. Use the
:guilabel:`Website` drop-down menu to restrict the publishing of this contact to one website (if
working on a database with multiple websites). Select one or more :guilabel:`Website Tags` to assist
in filtering published customers on the `/customers` website page. Select an :guilabel:`Industry`
for this contact from the drop-down menu. Use the :guilabel:`SLA Policies` field to assign a
*Helpdesk* SLA policy to this contact.
```